src/DataValidation.py

```diff
@@ -170,6 +173,11 @@ def checkItemsThatShouldBeRequired():
                 if isinstance(location_requires, str):
                     if '|{}|'.format(item["name"]) in location_requires:
                         raise ValidationError("Item %s is required by location %s, but the item is not marked as progression." % (item["name"], location["name"]))
+                    if itemValues and 'ItemValue' in location_requires:
+                        for value in re.findall(r'\{ItemValue\(([^:]*)\:[^)]+\)\}', location_requires):
+                            if value.lower().strip() in itemValues:
+                                raise ValidationError(f"Item {item['name']} has a value for the '{value}' value required by location '{location['name']}', but the item is not marked as progression.")

```

Comment:
Not sure how I feel about this.  I'd much rather rely entirely on `checkIfEnoughItemsForValue` for checking there's enough progression value.  

Right now, the main thing this validation does is say "5 Rupees" cannot be a filler item.